file.go

```diff
@@ -31,6 +30,7 @@ type File struct {
 	columnIndexes []format.ColumnIndex
 	offsetIndexes []format.OffsetIndex
 	rowGroups     []RowGroup
+	cfg           *FileConfig
```

Comment:
We used the word `config` rather than `cfg` in other types, what do you think about renaming this field to keep it consistent?

Reply:
+1, done